doc/neps/nep-0017-split-out-maskedarray.rst

```diff
@@ -0,0 +1,133 @@
+=======================
+Split Out Masked Arrays
+=======================
+
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Status: Deferred
```

Comment:
This should be Rejected. Deferred means *The NEP author or a core developer can assign the NEP this status when no progress is being made on the NEP.* --> that's not the case here.

Reply:
In the [template](https://github.com/numpy/numpy/blob/master/doc/neps/index.rst.tmpl), there are slots for `Accepted`, `Draft`, `Final`, and `Deferred`, it seems `Rejected` ones are not shown, while the `Deferred` ones appear under the `Defunct` title. Is that a bug?

Reply:
Seems inconsistent - the template should use all the possible statuses listed in http://www.numpy.org/neps/nep-0000.html I'd think